Update the description for DataGrid - Columns Hiding Priority (#83)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/GridColumnsHidingPriority/description.md

> When the **DataGrid** adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority). Columns with lower values are hidden first. 

Reviewer: ```suggestion
When the **DataGrid** adapts its layout to smaller screens, it hides columns based on the [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority). Columns with lower values are hidden first. 
```

> **DataGrid** automatically adapts its layout to screens of different widths. If screen width is not enough to display all columns, the widget hides them one by one. Set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true** to enable this feature.

Reviewer 2: The reader already read that in the previous demo description. Let's try start with

"When the DataGrid adapts its layout to small screen width, it hides columns based on [hidingPriority]()."

Then continue describing how this works. Finally, deescribe what's happening in the demo.

> When the **DataGrid** adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority). The lower a column's **hidingPriority**, the sooner the column will be hidden. For example, a column with priority 0 will be hidden first, then a column with priority 1, and so on.

Reviewer 3: I think the last two sentences can be merged into one shorter sentence. 

```suggestion
When the **DataGrid** adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority). Columns with lower values are hidden first. 
```

> In this demo, the `CustomerStoreCity` column is hidden in both vertical and horizontal orientation modes because its **hidingPriority** is 0. The `CustomerStoreState` and `OrderDate` columns have priorities 1 and 2. The **DataGrid** hides these columns after the `CustomerStoreCity` column, only when the screen orientation is vertical.

Reviewer: ```suggestion
In this demo, the `CustomerStoreCity` column is hidden in vertical and horizontal orientation modes because its **hidingPriority** is 0. The `CustomerStoreState` and `OrderDate` columns have a priority of 1 and 2. The **DataGrid** hides these columns after the `CustomerStoreCity` column, only when the screen orientation is vertical.
```

> When the DataGrid adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority).
> 
> The lower is this option's value, the sooner a column is hidden. For example, **DataGrid** hides the column whose **hidingPriority** equals *"0"* first, then the column with the priority *"1"* and so on.

Reviewer 2: ```suggestion
When the **DataGrid** adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority). The lower a column's **hidingPriority**, the sooner the column will be hidden. For example, a column with priority 0 will be hidden first, then a column with priority 1, and so on.
```

> In this demo, the `CustomerStoreCity` column has hiding priority *"0"* and is hidden in both vertical and horizontal orientation modes. The `CustomerStoreState` and `OrderDate` columns have priority *"1"* and *"2"* correspondingly. **DataGrid** hides these columns after the `CustomerStoreCity` column when the screen orientation is vertical.

Reviewer 2: ```suggestion
In this demo, the `CustomerStoreCity` column is hidden in both vertical and horizontal orientation modes because its **hidingPriority** is 0. The `CustomerStoreState` and `OrderDate` columns have priorities 1 and 2. The **DataGrid** hides these columns after the `CustomerStoreCity` column, only when the screen orientation is vertical.
```
